start_esp/start_esp.py

```diff
@@ -366,6 +368,8 @@ def fetch_service_config(args):
                     logging.error("[ESP] Service name is not specified");
                 sys.exit(3)
 
+            args.services = args.service.split('|')
```

Comment:
I think args.services should already be assigned.  This line is not needed here, I think

Reply:
It is needed because if `args.service` is not set, and it's fetched from the metadata server, then it's only set in line 361. I suppose this could be moved closer to that to make the association clearer.

Reply:
Please don't do too many code refactory.  I don't want to break most of users that only using one service